backend/api/views/parentadvice.py

```diff
@@ -0,0 +1,85 @@
+from flask import Blueprint
+from flask import request
+from api.core import (
+    create_response,
+    serialize_list,
+    logger,
+    invalid_model_helper,
+    admin_route,
+)
+from api.models import db, ParentAdvice
+
+parent_advice = Blueprint("parent_advice", __name__)
+
+
+@parent_advice.route("/parent_advice", methods=["POST"])
+@admin_route
+def create_parent_advice(user_id):
+    user_data = request.get_json()
+    if invalid_model_helper(user_data, ["text"]):
+        return create_response(
+            message="Missing parent advice text", status=422, data={"status": "fail"}
+        )
+    parent_advice = ParentAdvice(user_data["text"])
+
+    db.session.add(parent_advice)
+    db.session.commit()
+
+    return create_response(message="Successfully created a parent advice", status=200)
+
+
+@parent_advice.route("/edit_parent_advice", methods=["POST"])
+@admin_route
+def edit_parent_advice(user_id):
+    user_data = request.get_json()
+    if invalid_model_helper(user_data, ["text", "id"]):
+        return create_response(
+            message="Missing parent advice id or text",
+            status=422,
+            data={"status": "fail"},
+        )
+    parent_advice = ParentAdvice.query.get(user_data["id"])
+    if parent_advice is None:
+        return create_response(
+            message="Parent advice not found", status=422, data={"status": "fail"}
+        )
+
+    parent_advice.text = user_data["text"]
```

Comment:
o does this do the change?

Reply:
lol yup that does the change

Reply:
i'm gonna have to read the BE magic after the storm dies down